### ISSUE TITLE
Testers for Android Auto support wanted

### ISSUE THREAD

**Issue by y20k** [y20k/transistor#148]
One of the upcoming releases of Transistor will feature basic support for [Android Auto](https://www.android.com/auto/). 

My problem is: I do not have a car. So if **anyone with an actual Android Auto equipped car** wants to test the new feature... here is an APK:

## Download

* [transistor-app-test-2.3.4-01.apk](https://www.dropbox.com/s/tqemseij9b9k3kt/transistor-app-test-2.3.4-01.apk?dl=0) (based on this [commit](https://github.com/y20k/transistor/tree/192501e89763db1296fc3661420f6c22f90d22c5))

## Test using your phone
If you want to test Auto support on your phone you need to enable **Developer Mode** and **Unknown Sources**

* see: [Testing apps for phone screen](https://developer.android.com/training/auto/testing/index.html#phone) 

## Test using your car
Sorry. I am not sure how to test this in an actual car. You probably need to activate **Unknown Sources** somehow. Because by default Android Auto only runs / displays Apps from the Play Store.

## Screenshots Android Auto
![2017-08-30 14 30 55](https://user-images.githubusercontent.com/9103935/29872883-aeb3f518-8d91-11e7-9331-a1c6649ec73d.png) ![2017-08-30 14 28 25](https://user-images.githubusercontent.com/9103935/29872875-a6e4d7bc-8d91-11e7-94f2-c34bdda7f3c7.png) ![2017-08-30 14 29 19](https://user-images.githubusercontent.com/9103935/29872895-b8d1d114-8d91-11e7-865b-0207b9fa68e2.png) ![2017-08-30 14 28 45](https://user-images.githubusercontent.com/9103935/29872911-cbabaed6-8d91-11e7-9f9a-a45eed3625b1.png)

## Screenshots Android Wear

![screenshot_1504095986b](https://user-images.githubusercontent.com/9103935/29872845-80c71bee-8d91-11e7-8cef-12f6f86d5b49.png) ![screenshot_1504095956b](https://user-images.githubusercontent.com/9103935/29872843-80c38b28-8d91-11e7-9339-b833ae30fcb2.png) ![screenshot_1504095930b](https://user-images.githubusercontent.com/9103935/29872844-80c40c7e-8d91-11e7-9e2f-1335e3d368db.png)

**Comment by c00ller** [comment]
Hi y20k, great to see that you've got it integrated in the Android Auto phone app! Sorry, but my car doesn't have an official Android Auto supported screen, so I can't test that out. Anyway, I tried downloading the test apk, which from the name says it should be version 2.3.4. I downloaded it and installed it, but the app version still says 2.3. Is this right? Also, when I opened Android Auto on my phone, went to the media app selection screen, I didn't see Transistor in the list of apps :( Is the download link correct, or am I doing something wrong?
